erpnext/patches/v14_0/migrate_existing_lead_notes_as_per_the_new_format.py

```diff
@@ -16,7 +16,7 @@ def execute():
 
 		for d in records:
 			if strip_html(cstr(d.notes)).strip():
-				doc = frappe.get_doc(doctype, d.name)
+				doc = frappe.get_doc({"doctype":doctype, "name":d.name})
```

Comment:
This won't fix the issue. This is incorrect logic in fact.

The code here is attempting to load an existing doc, the change will init a new doc.